internal/kafka/internal/handlers/validation.go

```diff
@@ -95,18 +96,18 @@ func ValidateCloudProvider(kafkaService *services.KafkaService, kafkaRequest *db
 
 func ValidateKafkaUpdateFields(kafkaUpdateRequest *private.KafkaUpdateRequest) handlers.Validate {
 	return func() *errors.ServiceError {
-		if stringNotSet(&kafkaUpdateRequest.StrimziVersion) &&
-			stringNotSet(&kafkaUpdateRequest.KafkaVersion) &&
-			stringNotSet(&kafkaUpdateRequest.KafkaIbpVersion) &&
-			stringNotSet(&kafkaUpdateRequest.KafkaStorageSize) {
+		if !stringSet(&kafkaUpdateRequest.StrimziVersion) &&
+			!stringSet(&kafkaUpdateRequest.KafkaVersion) &&
+			!stringSet(&kafkaUpdateRequest.KafkaIbpVersion) &&
+			!stringSet(&kafkaUpdateRequest.KafkaStorageSize) {
```

Comment:
this would be more readable as:
```go

if !(stringSet(&kafkaUpdateRequest.StrimziVersion) ||
	 stringSet(&kafkaUpdateRequest.KafkaVersion) ||
	 stringSet(&kafkaUpdateRequest.KafkaIbpVersion) ||
	 stringSet(&kafkaUpdateRequest.KafkaStorageSize) {
```

Reply:
hwo about using a variadic function, e.g. `func multiStringSet(strings ...string)`?

Reply:
@pb82 - it would probably be nice, but there is an overhead of constructing the slice of kafkaUpdateRequest params. are you ok to go with Massimo's suggestion?

Reply:
Yes, that's fine too.

Reply:
I just successfully ran the tests locally with the latest changes. Merging this PR